src/app/project/project-documents/project-documents.component.ts

```diff
@@ -125,13 +125,12 @@ export class ProjectDocumentsComponent implements OnInit, OnDestroy {
       .takeUntil(this.ngUnsubscribe)
       .subscribe((res: any) => {
         if (res) {
-          if (res.documents[0].data.meta && res.documents[0].data.meta.length > 0) {
-            this.tableParams.totalListItems = res.documents[0].data.meta[0].searchResultsTotal;
-            this.documents = res.documents[0].data.searchResults;
-          } else {
-            this.tableParams.totalListItems = 0;
-            this.documents = [];
-          }
+					const documents = res.documents?.documents[0] || [];
```

Comment:
Okay this looks to be just a bug in the diff presentation on Github. Please disregard my previous comments about indentation